library/src/main/java/asia/ivity/android/marqueeview/MarqueeView.java

```diff
@@ -269,7 +274,6 @@ public void onAnimationEnd(Animation animation) {
                     reset();
                     return;
                 }
-                startTextFieldAnimation();
```

Comment:
Why this change?

Reply:
This makes sure once the Animation comes to an end and the cycle showing the full text and then reverting the text back to normal state runs once and does not repeat. By default the library repeats the Animation again once it comes to an end in a forever loop manner.

Reply:
So it seems that this was reverted--was that intentional?

Reply:
Ooh reverting this change.